src/chi/components/dropdown/dropdown.scss

```diff
@@ -0,0 +1,211 @@
+
+    &__trigger {
+
+      &:not([data-position^="left"]) {
+
+        &:after {
+          background-image: url(../html-select/icon-chevron-thin-down.svg);
+          background-position: center center;
+          background-repeat: no-repeat;
+          background-size: .75rem .75rem;
+          content: '';
+          display: block;
+          height: 1rem;
+          margin-left: 0.75rem;
+          width: 0.75rem;
+        }
+
+        &.-large:after {
+          height: 1.5rem;
+          margin-left: 1rem;
+        }
+
+        &.-small:after {
+          background-size: .5rem .5rem;
+          height: 1rem;
+          margin-left: 0.5rem;
+        }
+
+        &[data-position^="top"]:after {
+          transform: rotate(-180deg);
+        }
+
+        &[data-position^="right"]:after {
+          transform: rotate(-90deg);
+        }
+
+        &.-animate {
+          &:after {
+            transition: transform 0.2s ease-in-out;
+          }
+
+          &.-active {
+            &:after {
+              transform: rotate(180deg);
+            }
+
+            &[data-position^="right"]:after {
+              transform: rotate(90deg);
+            }
+
+            &[data-position^="top"]:after {
+                transform: rotate(0deg);
+            }
+          }
+        }
+      }
+
+      &[data-position^="left"] {
+
+        &:before {
+          background-image: url(../html-select/icon-chevron-thin-down.svg);
+          background-position: center center;
+          background-repeat: no-repeat;
+          background-size: .75rem .75rem;
+          content: '';
+          height: 1rem;
+          margin-right: 0.75rem;
+          transform: rotate(90deg);
+          width: 0.75rem;
+        }
+
+        &.-large:before {
+          height: 1.5rem;
+          margin-right: 1rem;
+        }
+
+        &.-small:before {
+          background-size: .5rem .5rem;
+          height: 1rem;
+          margin-right: 0.5rem;
+        }
+
+        &.-animate {
+          &:before {
+            transition: transform 0.2s ease-in-out;
+          }
+
+          &.-active:before {
+            transform: rotate(270deg);
+          }
+        }
+      }
+    }
+
+    &__menu {
+      background-color: set-color(white);
+      border: 0.0625rem solid set-color(grey, 30);
+      border-radius: $border-radius;
+      box-shadow: 0 1px 4px 0 rgba(set-color(black), 0.1);
+      display: none;
+      font-size: $text;
+      margin: 0;
+      padding: 0.5rem 0;
+
+      &.-active {
+        display: block;
+      }
+
+      & > a.m-dropdown__menu-item {
+        color: inherit;
+
+        &:hover {
+          text-decoration: none;
+        }
+
+        &:hover:not(.-disabled):not(.-active),
+        &.-hover:not(.-disabled):not(.-active),
+        &:focus:not(.-disabled):not(.-active),
+        &.focus:not(.-disabled):not(.-active)
+        {
+          background-color: set-color(grey, 20);
+        }
+      }
+
+      & > .m-dropdown__menu-item {
```

Comment:
@dani-cl-madrid try to remove the  > .m-dropdown__menu and set it to &-item in the &__menu ancestor